package/libyui-ncurses.changes

```diff
@@ -94,7 +101,7 @@ Mon Aug  7 13:21:35 UTC 2017 - mvidner@suse.com
 Tue Jul  4 07:58:10 UTC 2017 - mfilka@suse.com
 
 - bnc#1047145
-  - patch to make the package buildable by gcc7 (by werner@suse.com) 
+  - patch to make the package buildable by gcc7 (by werner@suse.com)
```

Comment:
Your editor killed all the old trailing whitespace. I like that; but I am not sure if the OBS team will also like the fact that the diff is so large.

Reply:
Yeah, I've decided to leave it. See no reason in drugging these forever.